### PR TITLE
add support for uml diagram in dark mode

### PR DESCRIPTION
**Pull request by huber-florian**
This PR adds support for UML diagrams in dark mode based on system settings.

Bildschirm­foto 2022-12-05 um 22 12 05

Bildschirm­foto 2022-12-05 um 22 12 34

### CODE REVIEW

File: Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift [hunk 1]
```diff
@@ -15,10 +15,17 @@ protocol ProblemStatementPart {
...
     var url: URL? {
         var url = URL(string: "https://artemis-staging.ase.in.tum.de/api/plantuml/png")
-        url?.append(queryItems: [URLQueryItem(name: "plantuml", value: text)])
+
+        if colorScheme == .light {
+            url?.append(queryItems: [URLQueryItem(name: "plantuml", value: text), URLQueryItem(name: "useDarkTheme", value: "false")])
+        } else {
+            url?.append(queryItems: [URLQueryItem(name: "plantuml", value: text), URLQueryItem(name: "useDarkTheme", value: "true")])
+        }
+
```
Comment (botrufus) [inline Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift]:
```suggestion
url?.append(queryItems: [URLQueryItem(name: "plantuml", value: text), URLQueryItem(name: "useDarkTheme", value: String(colorScheme != .light))])
```

**Review by botrufus — APPROVED** [review]
Looks good, just a small suggestion to shorten the code of the computed url.

File: Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift (hunk 1 above)
Comment (botrufus) [reply Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift]: can also do colorScheme == .dark if .dark exists

Comment (huber-florian) [reply Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift]: Great idea. Thanks.

**Review by pal03377 — APPROVED** [review]
Very nice!

**Pull request merged**